src/Dockerfile

```diff
@@ -39,22 +39,23 @@ COPY crontab.txt /crontab.txt
 # Add PADD to the container, too.
 ADD --chmod=0755 https://raw.githubusercontent.com/pi-hole/PADD/${PADD_BRANCH}/padd.sh /usr/local/bin/padd
 
-# download a the main repos from github
-RUN git clone --depth 1 --single-branch --branch ${WEB_BRANCH} https://github.com/pi-hole/web.git /var/www/html/admin && \
-    git clone --depth 1 --single-branch --branch ${CORE_BRANCH} https://github.com/pi-hole/pi-hole.git /etc/.pihole ;\
-    # Download the latest version of pihole-FTL for alpine:
-    if   [ "$TARGETPLATFORM" = "linux/amd64" ];    then FTLARCH=amd64; \
+
+# Download the latest version of pihole-FTL for alpine:
+RUN if   [ "$TARGETPLATFORM" = "linux/amd64" ];    then FTLARCH=amd64; \
     elif [ "$TARGETPLATFORM" = "linux/386" ];      then FTLARCH=386; \
     elif [ "$TARGETPLATFORM" = "linux/arm/v6" ];   then FTLARCH=armv6; \
     elif [ "$TARGETPLATFORM" = "linux/arm/v7" ];   then FTLARCH=armv7; \
     # Note for the future, "linux/arm6/v8" is not a valid value for TARGETPLATFORM, despite the CI platform name being that.
```

Comment:
Maybe move and change that comment to a link with all the valid TARGETPLATFORM values?

Reply:
I'm just going to remove the comment.

Reply:
Do you want to do anything with this comment? Remove? Change?